functions.js

```diff
@@ -0,0 +1,214 @@
+//1a==========================
+const allTvNames = inventory.map((item) => {
+    return item.name
+})
+console.log(allTvNames)
+
+
+//1b==========================
+const soldOutTvs = inventory.filter((item) => {
+    return item.sold === item.originalStock
+})
+console.log(soldOutTvs)
+
+
+//1c==========================
+const ambiLightTvs = inventory.filter((item) => {
+    return item.options.ambiLight
+})
+console.log(ambiLightTvs)
+
+
+//1d==========================
+const priceSort = inventory.sort((a, b) => {
+    return a.price - b.price
+})
+console.log(priceSort)
+
+
+//2a========================================================================================================
+const marketing = document.getElementById('marketing')
+
+function amountOfTvsSold(list) {
+    let amount = 0
+    list.map((item) => {
```

Comment:
Als je niets met het resultaat van de `map` doet, dan kun je beter een forEach gebruiken, zodat er niet in memory een array wordt opgebouwd
```suggestion
    list.forEach((item) => {
```